### ISSUE TITLE
Wordpress on GAE Flexible Cannot Change or upload images

### ISSUE THREAD

**Issue by meersant** [GoogleCloudPlatform/php-docs-samples#503]
I am trying to connect my images to google storage buket, but getting error with google app engine plugin and google media storage plugin.

**Comment by elharo** [comment]
Which error are you getting? Do you have a screenshot? Can you locate the related entry in your Stackdriver logs?

**Comment by meersant** [comment]
gcs plugin is installed and activated and the issue got resolved.

I have a problem with wp-all-import plugin.
How to change the public folder permission in Google App Engine Flexible ?

I am unable to use WP-ALL-IMPORT Plugin.
I get " Uploads folder gs://gcs-media-uploads/1/wpallimport/logs must be writable” 

I also tried WP ALL IN ONE MIGRATION Plugin. I get  
"All in One WP Migration is not able to create /app/wordpress/wp-content/ai1wm-backups folder. You will need to create this folder and grant it read/write/execute permissions (0777) for the All in One WP Migration plugin to function properly." 

Can i get help?
